net/pfSense-pkg-Avahi/files/usr/local/pkg/avahi.xml

```diff
@@ -3,12 +3,12 @@
 <?xml-stylesheet type="text/xsl" href="../xsl/package.xsl"?>
 <packagegui>
 	<copyright>
-<![CDATA[
+	<![CDATA[
 /*
  * avahi.xml
  *
  * part of pfSense (https://www.pfsense.org)
- * Copyright (c) 2009-2015 Rubicon Communications, LLC (Netgate)
+ * Copyright (c) 2018 Denny Page
```

Comment:
Don't remove Rubicon Copyright

Reply:
I didn't remove the copyright--in effect it wasn't there to begin with. This package is a complete ground up writing, unrelated to the prior package. This particular file was initialized by copying and modifying lldld.xml.